helm_deploy/cla-public/values-staging.yaml

```diff
@@ -18,13 +15,23 @@ ingress:
 
 # Lists don't deep merge, so this list of envVars overrides anything defined in an earlier values file
```

Comment:
This is no longer true, right?

Reply:
unneeded comments have now been removed.